editors/vscode/LICENSE

```diff
@@ -0,0 +1,31 @@
+Copyright Brent Yorgey 2021-2022
```

Comment:
Makes more sense to put your name, I think.

Reply:
It's just a copy-paste of the Microsoft template with generated TextMate grammar and keywords generated from the swarm binary, so I don't think it matters. :smile:

If we ever want to change the license for some reason, keeping it simple and not making it look like you have to get my signed permission could save us some headache. Licenses are complicated and I dread lawyers. :fearful:

Reply:
OK, fair enough!